floodviz/templates/hydrograph.html

```diff
@@ -3,32 +3,84 @@
 
 <head>
 	<meta charset="UTF-8">
-	<link rel="stylesheet" type="text/css" href="{{ url_for('static', filename='bower_components/bootstrap/dist/css/bootstrap.css') }}" />
+	<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no">
+	<link rel="stylesheet" type="text/css"
+		  href="{{ url_for('static', filename='bower_components/bootstrap/dist/css/bootstrap.css') }}"/>
 	<link rel="stylesheet" type="text/css" href="{{ url_for('static', filename='css/style.css') }}"/>
-	<link rel="stylesheet" type="text/css" href="{{ url_for('static', filename='css/hydrograph.css') }}"/>
+	<link rel="stylesheet" type="text/css" href="{{ url_for('static', filename='css/map.css') }}" />
+	<link rel="stylesheet" type="text/css" href="{{ url_for('static', filename='css/hydrograph.css') }}" />
 	<title>Flood Viz</title>
 
+	<script type="text/javascript">
```

Comment:
You should ask Jordan whether we want google analytics in these embeddable pages. If the answer is yes, then it will be more DRY to put this script in a separate file in templates (maybe called google_analytices.html) and use the JINJA template tag {% include %}